pkg/printers/internalversion/describe.go

```diff
@@ -2982,6 +2983,128 @@ func describeHorizontalPodAutoscaler(hpa *autoscaling.HorizontalPodAutoscaler, e
 	})
 }
 
+// VerticalPodAutoscalerDescriber generates information about a vertical pod autoscaler.
+type VerticalPodAutoscalerDescriber struct {
+	clientset.Interface
+	external externalclient.Interface
+}
+
+// Describe describes the vertical pod autoscaler object.
+func (d *VerticalPodAutoscalerDescriber) Describe(namespace, name string, describerSettings printers.DescriberSettings) (string, error) {
+	vpa, err := d.external.AutoscalingV2beta1().VerticalPodAutoscalers(namespace).Get(name, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+
+	var events *api.EventList
+	if describerSettings.ShowEvents {
+		events, _ = d.Core().Events(namespace).Search(legacyscheme.Scheme, vpa)
+	}
+
+	internalVpa := &autoscaling.VerticalPodAutoscaler{}
+	if err := legacyscheme.Scheme.Convert(vpa, internalVpa, nil); err != nil {
+		return "", err
+	}
+
+	return describeVerticalPodAutoscaler(internalVpa, events, d)
```

Comment:
You only use internal vpa. Therefore you don't need external client or conversion.